tests/repegAndSpread.ts

```diff
@@ -334,6 +345,31 @@ describe('repeg and spread amm', () => {
 		const midPrice = (convertToNumber(bid) + convertToNumber(ask)) / 2;
 
 		console.log(convertToNumber(oraclePriceData.price), midPrice);
+		console.log(
+			'getBankAssetValue:',
+			clearingHouseUser.getBankAssetValue().toString()
+		);
+
+		const effectiveLeverage = calculateEffectiveLeverage(
+			prepegAMM.baseSpread,
+			prepegAMM.quoteAssetReserve,
+			prepegAMM.terminalQuoteAssetReserve,
+			prepegAMM.pegMultiplier,
+			prepegAMM.netBaseAssetAmount,
+			markPrice,
+			prepegAMM.totalFeeMinusDistributions
+		);
+		const inventoryScale = calculateInventoryScale(
+			prepegAMM.netBaseAssetAmount,
+			prepegAMM.baseAssetReserve,
+			prepegAMM.minBaseAssetReserve,
+			prepegAMM.maxBaseAssetReserve
+		);
+
+		console.log('inventoryScale:', inventoryScale);
+		console.log('effectiveLeverage:', effectiveLeverage);
+		assert(Math.min(effectiveLeverage, 5) == 5); // lol
+		assert(inventoryScale == 0.034835);
```

Comment:
is there a way to back test these against the actual values in the contract - maybe compute the change in the long/short spreads before / after the inventory scale - might not be that important idk - depends on how often we'll be using these fcns in the sdk